### PR TITLE
Migrated authentication to SQL database

### PR DESCRIPTION
**Pull request by JerryKu**
# In this PR

- Authentication now handled by Postgres and Sequelize

# TODO
- Migrate activities to SQL database and change api call on front end
- Facebook authentication still not working
- Add ability to rate and like activities

### CODE REVIEW

File: client/dist/index.html [hunk 1]
```diff
@@ -10,6 +10,7 @@
 		<link rel="stylesheet" href="./style.scss">
 		<link href="https://fonts.googleapis.com/css?family=Rubik" rel="stylesheet">
 		<link rel="stylesheet" href="https://unpkg.com/flickity@2/dist/flickity.min.css">
+		<link rel="stylesheet" href="https://unpkg.com/flickity@2/dist/flickity.min.css">
```
Comment (andrewchen44) [inline client/dist/index.html]: why is this here twice?

File: server/config/config.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+  "development": {
+    "username": "pohifnlb",
+    "password": "9dHCnAN3kglE0rXZ4qou1Qf2jQlxCc-A",
+    "database": "pohifnlb",
+    "host": "pellefant.db.elephantsql.com",
+    "dialect": "postgres"
+  },
```
Comment (andrewchen44) [inline server/config/config.json]: this whole config file should have been in git ignore, you are going to create a new file with new credentials because the whole world can see this now

File: server/index.js [hunk 3]
```diff
@@ -1,15 +1,43 @@
...
 
 const app = express();
 const port = process.env.PORT || 8080;
+//import models
+app.set('models', require('./models'));
+const models = app.get('models');
+//load passport strategies
+require('../config/passport')(passport, models.authentications, models.users);
+
+models.sequelize.sync().then(function(){
```
Comment (andrewchen44) [inline server/index.js]: shouldn't connect directly to the database like this only when necessary (like when it hits the login route)

File: client/dist/index.html (hunk 1 above)
Comment (JerryKu) [reply client/dist/index.html]: Dunno how that got added. removed.

File: server/index.js (hunk 3 above)
Comment (JerryKu) [reply server/index.js]: Commented it out for when we need to sync the database models

File: server/config/config.json (hunk 2 above)
Comment (JerryKu) [reply server/config/config.json]: Removed file, should be gone when this branch is deleted.
